content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -21,6 +21,12 @@ The following instructions assume you have installed Helm and use it to install
 To install Kiali without using Helm, follow the [Kiali installation instructions](https://www.kiali.io/gettingstarted/).
 {{< /idea >}}
 
+### Create a Secret
+
+{{< idea >}}
+If you plan on installing Kiali using the `istio-demo.yaml` or `istio-demo-auth.yaml` file as described in the [Istio Quick Start Installation Steps](https://istio.io/docs/setup/kubernetes/quick-start/#installation-steps) then a default secret will be created for you with a username of `admin` and passphrase of `admin`. You can therefore skip this section.
```

Comment:
Drop the 'https://istio.io' from the URL.

Reply:
done.